rebasehelper/cli.py

```diff
@@ -200,6 +200,13 @@ def build_parser():
             help=argparse.SUPPRESS
         )
         parser.add_argument(
+            "--pure-html-output",
+            dest="pure_html_output",
+            action="store_true",
+            default=False,
+            help="generate pure html when using html output tool"
```

Comment:
In that case, this option can be removed totally, as no javascript is being generated nor used.